store/listings/views.py

```diff
@@ -1,3 +1,58 @@
-from django.shortcuts import render
+from typing import Dict
 
-# Create your views here.
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers, viewsets
+
+from .models import Listing
+
+
+def swagger_docs(methods: Dict):
```

Comment:
Move `swagger_docs` func to `store/store/common/utils.py`

Reply:
Done